src/legacy/core_plugins/console/np_ready/public/application/containers/editor/legacy/subscribe_console_resize_checker.ts

```diff
@@ -16,9 +16,14 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+import { ResizeCheckerClass } from 'src/plugins/kibana_utils/public';
```

Comment:
You can just import `ResizeChecker` directly, as it's stateless - you don't have to pass it in as a dependency

Reply:
I explained why below.